components/automate-ui/package.json

```diff
@@ -41,6 +41,7 @@
     "@ngrx/store": "^8.3.0",
     "@ngrx/store-devtools": "^8.3.0",
     "@nguniversal/express-engine": "^8.1.1",
+    "chromedriver": "^78.0.1",
```

Comment:
I am not that familiar with the usage of chromedriver, but this change makes me wonder:
If this line was just bumping a version that was already present in the file, that would seem fine. However, _adding_ a new dependency when none was needed before makes me nervous.

Reply:
(Also it now becomes a small maintenance burden since that version will change frequently.)

Reply:
That's fair-- I followed these steps to get things running again 
https://stackoverflow.com/a/58105741

Reply:
That's fair-- I followed these steps to get things running again 
https://stackoverflow.com/a/58105741

I mostly was trying to unblock myself-- happy to do something different if you have an idea!

Reply:
I deleted the section for `webdriver-manager` from `package-lock.json` and then did `npm install` and it upgraded me to the latest version of `webdriver-manager`:

```
diff --git a/components/automate-ui/package-lock.json b/components/automate-ui/package-lock.json
index 6c31f909..b3f23d15 100644
--- a/components/automate-ui/package-lock.json
+++ b/components/automate-ui/package-lock.json
@@ -10621,9 +10621,9 @@
           "dev": true
         },
         "webdriver-manager": {
-          "version": "12.1.6",
-          "resolved": "https://registry.npmjs.org/webdriver-manager/-/webdriver-manager-12.1.6.tgz",
-          "integrity": "sha512-B1mOycNCrbk7xODw7Jgq/mdD3qzPxMaTsnKIQDy2nXlQoyjTrJTTD0vRpEZI9b8RibPEyQvh9zIZ0M1mpOxS3w==",
+          "version": "12.1.7",
+          "resolved": "https://registry.npmjs.org/webdriver-manager/-/webdriver-manager-12.1.7.tgz",
+          "integrity": "sha512-XINj6b8CYuUYC93SG3xPkxlyUc3IJbD6Vvo75CVGuG9uzsefDzWQrhz0Lq8vbPxtb4d63CZdYophF8k8Or/YiA==",
           "dev": true,
           "requires": {
             "adm-zip": "^0.4.9",
```

I don't know if that will work on CI but it made it so I can run the tests locally.

Reply:
npm does not appear to have a straightforward way to update transitive dependencies, unfortunately.

Reply:
@phiggins wanna close this and open a PR?